src/app/domain/geolocation/tracking/index.js

```diff
@@ -70,10 +70,8 @@ export const startTracking = async () => {
         smallIcon: 'mipmap/ic_stat_ic_notification'
       }
     })
-    BackgroundGeolocation.start(() => {
-      Log('Tracking started')
-      storeData(StorageKeys.ShouldBeTrackingFlagStorageAdress, true)
-    })
+    await BackgroundGeolocation.start()
```

Comment:
Could you just set back the `Tracking started` log? It is sometimes useful to know when it was started